src/runtime/tests/schema-test.ts

```diff
@@ -443,4 +443,25 @@ describe('schema', () => {
     assert.strictEqual(schema.fields.age.type, 'Number');
     assert.strictEqual(schema.fields.custom.type, 'Bytes');
   });
+
+  it('parses annonymous schemas', async () => {
```

Comment:
This schema is named (X) but has no fields, is it anonymous or empty?

Reply:
Schema has no names. 'X' is an alias for referencing this schema.

Reply:
Sounds good